tests/conftest.py

```diff
@@ -57,3 +61,25 @@ def dvc(repo_dir, git):
         yield dvc
     finally:
         dvc.scm.git.close()
+
+
+here = os.path.abspath(os.path.dirname(__file__))
+
+user = "user"
+key_path = os.path.join(here, "{0}.key".format(user))
+
+
+@pytest.yield_fixture()
+def ssh():
```

Comment:
You could use `ssh_server` fixture in this fixture, so you don't duplicate code :)

Reply:
Cool. DONE